doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -202,7 +202,13 @@ nRF5340 Audio
 nRF Desktop
 -----------
 
-|no_changes_yet_note|
+* Added handling of :c:struct:`hid_report_subscriber_event` to :ref:`nrf_desktop_hid_forward`.
+  Handling the :c:struct:`hid_report_subscriber_event` allows to take into account the maximum number of reports with different ID, which can be processed by the subscriber (:c:member:`hid_report_subscriber_event.params.report_max`).
+* Added :ref:`nrf_desktop_hid_reportq`.
```

Comment:
```suggestion
* Added :ref:`nrf_desktop_hid_reportq` page.
```

Reply:
Whole module implementation was added since NCS 2.7

Reply:
Maybe section instead of page then.

Reply:
Sections also refers to documentation only. I want to inform that we added a brand new module here.

Reply:
We already used similar approach in the past. See https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/releases_and_maturity/releases/release-notes-2.4.0.html#nrf_desktop for an example